### PR TITLE
Implement a way to set custom biome definitions

### PR DESCRIPTION
**Pull request by JustTalDevelops**

### CODE REVIEW

File: minecraft/listener.go [hunk 1]
```diff
@@ -43,6 +43,9 @@ type ListenConfig struct {
 	// This field should not be edited during runtime of the Listener to avoid race conditions. Use
 	// Listener.AddResourcePack() to add a resource pack after having called Listener.Listen().
 	ResourcePacks []*resource.Pack
+	// BiomeDefinitions is a list of encoded biome definitions that the server may use. If these are nil,
+	// the default biome definitions will be used.
+	BiomeDefinitions []byte
```
Comment (Sandertv) [inline minecraft/listener.go:48]: I'd make this a map[string]interface{} and leave the NBT encoding to gophertunnel

Comment (JustTalDevelops) [reply minecraft/listener.go]: Changed.

File: minecraft/listener.go [hunk 2]
```diff
@@ -43,6 +43,9 @@ type ListenConfig struct {
 	// This field should not be edited during runtime of the Listener to avoid race conditions. Use
 	// Listener.AddResourcePack() to add a resource pack after having called Listener.Listen().
 	ResourcePacks []*resource.Pack
+	// BiomeDefinitions is a list of encoded biome definitions that the server may use. If these are nil,
+	// the default biome definitions will be used.
+	BiomeDefinitions map[string]interface{}
```
Comment (Sandertv) [inline minecraft/listener.go:48]: I think it makes more sense to put this in the GameData.

Comment (Sandertv) [reply minecraft/listener.go:48]: Would also just call it `Biomes` there.

Comment (JustTalDevelops) [reply minecraft/listener.go:48]: Isn't GameData only meant for data in the start game packet?

Comment (Sandertv) [reply minecraft/listener.go:48]: Not necessarily, although now that you mention it, it does seem to complicate things a bit if we have it contain data of other packets too... I'd just call it `Biomes` anyway and leave it here for now. That's all right.

Comment (JustTalDevelops) [reply minecraft/listener.go:48]: Alright, will do.

**Pull request merged**